src/opera/instance/node.py

```diff
@@ -80,16 +81,44 @@ def get_host(self):
     def _configure_requirements(self,
                                 source_operation: ConfigureInterfaceOperation,
                                 target_operation: ConfigureInterfaceOperation,
-                                verbose: bool, workdir: str):
+                                verbose: bool, workdir: str,
+                                validate: bool = False):
         for requirement in set(r.name for r in self.template.requirements):
             for relationship in self.out_edges[requirement].values():
                 relationship.run_operation(OperationHost.SOURCE, ConfigureInterfaceOperation.shorthand_name(),
-                                           source_operation, verbose, workdir)
+                                           source_operation, verbose, workdir, validate)
 
         for requirement_dependants in self.in_edges.values():
             for relationship in requirement_dependants.values():
                 relationship.run_operation(OperationHost.TARGET, ConfigureInterfaceOperation.shorthand_name(),
-                                           target_operation, verbose, workdir)
+                                           target_operation, verbose, workdir, validate)
+
+    def validate(self, verbose, workdir):
+        thread_utils.print_thread("  Validating {}".format(self.tosca_id))
+
+        # validate node's deployment
+        self.run_operation(OperationHost.HOST, StandardInterfaceOperation.shorthand_name(),
+                           StandardInterfaceOperation.CREATE, verbose, workdir, True)
+        self._configure_requirements(ConfigureInterfaceOperation.PRE_CONFIGURE_SOURCE,
+                                     ConfigureInterfaceOperation.PRE_CONFIGURE_TARGET, verbose, workdir)
```

Comment:
I think you're missing a `True` as the last parameter for the `validate` argument?

Reply:
Aha yes, it's true. I've resolved this one.